src/frontend/scss/settings/_gradients.scss

```
@@ -0,0 +1,27 @@
+// Named gradients
+// ---------
+$neutral-gradient: linear-gradient(
+  to top,
+  r-color('silver') 0,
+  r-color('smoke') 13rem,
+  r-color('white') 14rem,
+  r-color('white')
+);
+$middle-gradient: linear-gradient(285deg, r-color('grey59') 3%, r-color('grey87') 98%);
+$dark-gradient: linear-gradient(285deg, r-color('grey32') 3%, r-color('grey59') 98%);
+$white-mask-gradient: (
+  linear-gradient(
+    to right,
+    r-color('white') 0%,
+    rgba(r-color('white'), 0.85) 15%,
+    rgba(r-color('white'), 0.15) 60%,
+    rgba(r-color('white'), 0) 70%
+  ),
+  linear-gradient(
+    to left,
+    r-color('white') 0%,
+    rgba(r-color('white'), 0.85) 15%,
+    rgba(r-color('white'), 0.15) 60%,
+    rgba(r-color('white'), 0) 70%
+  )
+);
```

We could make default gradients easily overridable if we suffix them with `!default`.

```suggestion
$neutral-gradient: linear-gradient(
  to top,
  r-color('silver') 0,
  r-color('smoke') 13rem,
  r-color('white') 14rem,
  r-color('white')
) !default;
$middle-gradient: linear-gradient(285deg, r-color('grey59') 3%, r-color('grey87') 98%) !default;
$dark-gradient: linear-gradient(285deg, r-color('grey32') 3%, r-color('grey59') 98%) !default;
$white-mask-gradient: (
  linear-gradient(
    to right,
    r-color('white') 0%,
    rgba(r-color('white'), 0.85) 15%,
    rgba(r-color('white'), 0.15) 60%,
    rgba(r-color('white'), 0) 70%
  ),
  linear-gradient(
    to left,
    r-color('white') 0%,
    rgba(r-color('white'), 0.85) 15%,
    rgba(r-color('white'), 0.15) 60%,
    rgba(r-color('white'), 0) 70%
  )
) !default;
```

---

@jbpenrath 

Added `!default` to all the gradients, schemes, palette and r_theme.

> I still wonder if we should not leave the responsibility to merge maps to the "child" project.

Yes, we could not include `overwrite` variables. But I think we should include the `recursive-map-merge` function so we could document how a child project should create a custom theming. 

> Further I wonder if we should not use !default also for palette and theme and let's manage merge from the "child" project.

The problem is that the `palette` and the `theme` are maps. They are only 2 variables, the `!default` would work if we transform each line of the `palette` and `theme` to a specific variable.

---

> @jbpenrath
> 
> Added `!default` to all the gradients, schemes, palette and r_theme.
> 
> > I still wonder if we should not leave the responsibility to merge maps to the "child" project.
> 
> Yes, we could not include `overwrite` variables. But I think we should include the `recursive-map-merge` function so we could document how a child project should create a custom theming.

👍 

> 
> > Further I wonder if we should not use !default also for palette and theme and let's manage merge from the "child" project.
> 
> The problem is that the `palette` and the `theme` are maps. They are only 2 variables, the `!default` would work if we transform each line of the `palette` and `theme` to a specific variable.

Bootstrap manages its variables as you describe. In our case as our color name variable are related to the tint itself, I don't think its relevant.

---

Ok